depends/hosts/linux.mk

```diff
@@ -1,7 +1,7 @@
 linux_CFLAGS=-pipe
 linux_CXXFLAGS=$(linux_CFLAGS)
 
-linux_release_CFLAGS=-O2
+linux_release_CFLAGS=-O1
 linux_release_CXXFLAGS=$(linux_release_CFLAGS)
 
 linux_debug_CFLAGS=-O1
```

Comment:
Ah, ``_GLIBCXX_DEBUG`` is documented at https://gcc.gnu.org/onlinedocs/libstdc++/manual/debug_mode.html